app/models/filter.rb

```diff
@@ -135,6 +141,21 @@ def expire_topbar_cache(sweeper)
     role.usergroups.each { |g| g.expire_topbar_cache(sweeper) }
   end
 
+  def disable_overriding!
+    self.override = false
+    self.save!
+  end
+
+  def inherit_taxonomies
+    inherit_taxonomies! unless self.override?
```

Comment:
A common thread among both of your reasonings is ["once we add this somewhere else..." " this endpoint should be present"](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).  

Within this PR at least , the method is irrelevant and actually takes more time to read (I jump from before_save to inherit_taxonomies, then to actual `inherit_taxonomies!` vs 'before_save -> inherit_taxonomies!`). 

In any case, I don't want to block the PR on a minor thing, so consider this my last reply to this issue